strings/README.md

```diff
@@ -3,3 +3,7 @@
 ### C or C++
 
 1. [Palindrome Check](c-or-cpp/palindrome.c)
+
+### C#
+Please use [.Net Finddle](https://dotnetfiddle.net/)
```

Comment:
Shouldn't it be " you can use any online IDE (for example [.net Finddle](https://dotnetfiddle.net/) ) to test them " ?

Reply:
Ok